### DISCUSSION TITLE
Switching colours

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1054] [General]
Hi
Is it possible to change these colours and how?
Thank you

![2023-04-04 09_49_59-Livre des parcours - MyTourbook - 23 3 0](https://user-images.githubusercontent.com/4082252/229726207-034c7bdc-5515-480c-a558-88eb2f99dd9f.jpg)

**Comment by wolfgang-ch** [comment]
> Is it possible to change these colours ...?

Not easily, maybe somewhere in the deep SWT CSS it could be customized. 

In earlier MT/Eclipse versions these triangles were even more "hidden" than now in the dark theme.
